release/signing_images.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# shellcheck source=common.sh
+# shellcheck disable=2162
+# shellcheck disable=2086
```

Comment:
sorry, just came back from a long holiday and busy with some other stuffs. Now back to this.

I recalled that i need to have these 2 disabled because it was complaining about it. Let me test and confirm it again.